fleet-management-service/src/test/java/cz/fi/muni/pa165/service/BeanMappingServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package cz.fi.muni.pa165.service;
+
+import cz.fi.muni.pa165.config.ServiceConfiguration;
+import cz.fi.muni.pa165.dto.*;
+import cz.fi.muni.pa165.entity.*;
+import cz.fi.muni.pa165.enums.Role;
+import cz.fi.muni.pa165.service.interfaces.BeanMappingService;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+import java.time.Year;
+import java.util.Date;
+
+/**
+ * @author Jozef Krcho
+ */
+@ContextConfiguration(classes = ServiceConfiguration.class)
+public class BeanMappingServiceImplTest extends AbstractTestNGSpringContextTests {
+
+    @Inject
+    private BeanMappingService beanMappingService;
+
+    @Test
+    public void testEmployeeMapping() {
```

Comment:
I think writing such a test is completely worthless. Sure, they have it in the [PA165 example project](https://github.com/fi-muni/PA165/blob/master/eshop-service/src/test/java/cz/fi/muni/pa165/service/BeanMappingServiceTest.java), but certainly they are not testing every entity. It resembled me a test of the anemic entity (entity with only getters and setters, but no business logic - like we have in our project), I'ts just lot of writing, but business logic is really tested. I would not bother by writing such tests, delete whole file and rely that implementation of Dozer is correct.